doc/api/cli.md

```diff
@@ -984,8 +984,9 @@ added: REPLACEME
 -->
 
 Loads environment variables from a file relative to the current directory.
-
-`NODE_OPTIONS` environment variable is not supported at the moment.
+Environment file supports `NODE_OPTIONS` as a key. If there is an
+environment variable named `NODE_OPTIONS`, environment variable is preferred
+over the file configuration.
```

Comment:
is overriding a better choice than merging? it's not clear to me which is better.

Reply:
I don't know. Since this is an experimental feature, I recommend following up with a different issue/pull request about any implementation strategies.